fab/environments.yml

```diff
@@ -111,6 +115,8 @@ production:
         concurrency: 8
       ucr_queue:
         concurrency: 4
+      email_queue:
+        concurrency: 4
```

Comment:
(just copying commit comment here for visibility)

two thoughts:

4 feels high to me since presumably sending emails is somewhat rare, timing is pretty random, and these emails should typically be quite quick. thoughts on reducing to 2? the memory used increases with each process so generally just trying to minimize unnecessary overhead.
I just added 4 more queues to hqcelery1 for UCR, so I think we should put these on hqcelery0. When I looked they were about the same.